docs/getting-started-guide.md

```diff
@@ -7,7 +7,7 @@
 $ make
 go build -o bin/rktlet ./cmd/server/main.go
 
-$ sudo ./bin/rktlet -v=4 -alsologtostderr
+$ sudo ./bin/rktlet -v=4 --alsologtostderr
```

Comment:
Heh, the init function of `pkg/util/logs` actually already does that, so looks like it's done already!

I'll fix the docs